src/NuGet.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!-- NOTE: Leave this file here and keep it in sync with list in dir.props. -->
+  <!-- The command-line doesn't need it, but the IDE does.                    -->
```

Comment:
What is "incorrect" about the command-line tool today. Is that something being tracked as a bug. This comment accurately reflects the status quo. If we want to merge these lists, we can open a bug on that and link to dnu/nuget bugs as appropriate. I'd rather not put the TODO here.